meson.build

```diff
@@ -2004,7 +2004,26 @@ if conf.get('HAVE_LIBCRYPTSETUP_PLUGINS') == 1
                         install_dir : libcryptsetup_plugins_dir)
         endif
 
-        if conf.get('HAVE_P11KIT') == 1
+        if conf.get('HAVE_P11KIT') == 1 and conf.get('HAVE_OPENSSL') == 1
+                        cryptsetup_token_systemd_pkcs11 = shared_library(
+                        'cryptsetup-token-systemd-pkcs11',
+                        cryptsetup_token_systemd_pkcs11_sources,
+                        include_directories : includes,
+                        link_args : ['-shared',
+                                     '-Wl,--version-script=' + cryptsetup_token_sym_path],
+                        link_with : [lib_cryptsetup_token_common,
+                                     libshared],
+                        dependencies : [libcryptsetup,
+                                        libp11kit,
+                                        libopenssl,
+                                        versiondep],
+                        link_depends : cryptsetup_token_sym,
+                        install_rpath : rootlibexecdir,
+                        install : true,
+                        install_dir : libcryptsetup_plugins_dir)
+        endif
```

Comment:
this seems unnecessary, just add the openssl dep unconditionally. cryptenroll has it anyway, there's no point in conditionalizing it here (also, please don't duplicate this many lines, you can build up deps list with various expressions quite easily, just look around in our meson files, there are plenty examples)